libraries/rush-lib/src/logic/setup/KeyboardLoop.ts

```diff
@@ -12,6 +12,8 @@ import { AlreadyReportedError, InternalError } from '@rushstack/node-core-librar
 const ANSI_ESCAPE_SHOW_CURSOR: string = '\u001B[?25l';
 const ANSI_ESCAPE_HIDE_CURSOR: string = '\u001B[?25h';
 
+type OptionalProperty<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>> & Partial<Pick<T, K>>;
```

Comment:
Unrelated change?

Reply:
This is to resolve the one typescript error that surfaced from the version bump- that there is a check later in the file for the existence of a required function of stdin's type, so I cast is as a type where that property is optional for that check